### ISSUE TITLE
Come up with software architecture

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#28]
## User story
1. As a stakeholder
2. I want to be able know which software architecture is used for the app
3. So that I can get a better picture of how the app is set up.

## Acceptance criteria
* A document describing the software architecture has been created.
* The document has been uploaded to the deliverables folder of sprint-02 in PDF-format.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
